docs/dev/isolation.md

```diff
@@ -45,3 +45,7 @@ TODO
 
 ## Multi-tenancy
 We do not yet support multi-tenancy, in the sense that different tenants of the same server get isolated performance guarantees. When we do support this, it will need to be documented here.
+
+## Preserving isolation across RPC calls
```

Comment:
There's a line "RPC: TODO: explain commit 8c993e0728508728c2d5cb0c4323728831d84890." which I think you'll want to delete now.

Reply:
Removed.